shell/app/node_main.cc

```diff
@@ -195,6 +258,12 @@ int NodeMain(int argc, char* argv[]) {
 
       gin_helper::Dictionary process(isolate, env->process_object());
       process.SetMethod("crash", &ElectronBindings::Crash);
+#if BUILDFLAG(IS_LINUX)
+      // These are needed so that process forked from this child process
+      // can connect to the crashpad handler process.
+      process.SetMethod("getCrashdumpSignalFD", &GetCrashdumpSignalFD);
+      process.SetMethod("getCrashpadHandlerPID", &GetCrashpadHandlerPID);
```

Comment:
Let's move these methods to a separate internal module, e.g. `process._linkedBinding('crashpad_support')`

Reply:
Addressed in https://github.com/electron/electron/pull/36460/commits/21b0e1ffadfa7590a9deb814c2bf368c612cb336